fastavro/_validation.pyx

```diff
@@ -353,18 +353,16 @@ cpdef validate(object datum, object schema, str field='',
     return bool(result)
 
 
-cpdef validate_many(records, schema, bint raise_errors=False, int stop_count=-1):
+cpdef validate_many(records, schema, bint raise_errors=True):
```

Comment:
Why was stop_count removed? If validating many items and you wanted to exit on the first error, this would allow for early exit. Or if you wanted to capture all errors or just a few, it allows for that situation. Thoughts?